config.json

```diff
@@ -244,7 +244,10 @@
     "restrict_to_login": false,
     "swatch_option_size": "22x22",
     "social_icon_placement_top": false,
-    "social_icon_placement_bottom": "bottom_none"
+    "social_icon_placement_bottom": "bottom_none",
+    "show_geotrust_ssl_seal": false,
+    "geotrust_ssl_seal_hostname": "",
```

Comment:
why not use `shopPath` instead ? Is there a reason for not using the store domain instead of offering a separate field ?